.github/workflows/reusable-create-collector-contrib-pull-request.yml

```
@@ -62,7 +62,13 @@ jobs:
           VERSION: ${{ inputs.jmx-metrics-version }}
         run: |
           message="Update the jmx-metrics version to $VERSION"
-          body="Update the jmx-metrics version to \`$VERSION\`."
+          # note that @open-telemetry/java-contrib-approvers cannot be used below
+          # because opentelemetrybot is not a member of the OpenTelemetry org,
+          # and so it cannot @ mention OpenTelemetry org groups
+          body="Update the jmx-metrics version to \`$VERSION\`.
+
+          cc @jack-berg @mateuszrzeszutek @trask
```

I know I've been awol for a bit, but feel free to add me.

```suggestion
          cc @jack-berg @mateuszrzeszutek @trask @rmfitzpatrick 
```